browser/widevine/widevine_utils.cc

```diff
@@ -72,21 +74,28 @@ void ClearWidevinePrefs(Profile* profile) {
 
 }  // namespace
 
-void EnableWidevineCdmComponent(content::WebContents* web_contents) {
-  DCHECK(web_contents);
-
+void EnableWidevineCdmComponent() {
   if (IsWidevineOptedIn())
     return;
 
   SetWidevineOptedIn(true);
   RegisterWidevineCdmComponent(g_brave_browser_process->component_updater());
 }
 
+void DisableWidevineCdmComponent() {
+  if (!IsWidevineOptedIn())
+    return;
+
+  SetWidevineOptedIn(false);
+  g_brave_browser_process->component_updater()->UnregisterComponent(
```

Comment:
random idea: maybe we we could physically erase the component from disk

Reply:
Unregistering does deleting also :)